content/21.nutraceuticals.md

```diff
@@ -70,80 +117,121 @@ In inflammatory situations such as an infection, zinc can regulate leukocyte imm
+A study in India has determined that COVID-19 fatality is higher in patients with severe COVID-19 and low serum 25-hydroxyvitamin D (mean level 6.2 ng/ml; 97% vitamin D deficient) levels versus asymptomatic non-severe patients with higher levels of vitamin D (mean level 27.9 ng/ml; 33% vitamin D deficient) [@doi:10.1038/s41598-020-77093-z]. 
+In the same study, vitamin D deficiency was associated with higher levels of inflammatory markers including IL-6, ferritin, and TNFa.
+Collectively, these studies add to a multitude of observational studies reporting potential associations between low levels of 25-hydroxyvitamin D and COVID-19 incidence and severity [@doi:10.3390/nu12051359; @doi:10.3390/nu12123773; @doi:10.1007/s40618-020-01370-x; @doi:10.1210/clinem/dgaa733; @doi:10.3390/nu12092757; @doi:10.3390/nu12092775; @doi:10.1080/07315724.2020.1826005; @doi:10.1001/jamanetworkopen.2020.19722; @doi:10.1016/j.intimp.2020.107001; @doi:10.1093/jn/nxaa332].
+
+Despite these studies establishing a link between vitamin D status with COVID-19 severity, an examination of the UK Biobank did not support this thesis [@doi:10.1016/j.dsx.2020.04.050; @doi:10.1007/s00394-020-02372-4].
+These analyses examined 25-hydroxyvitamin D concentrations  and SARS-CoV-2 positivity and COVID-19 mortality in over 340,000 UK Biobank participants.
+However, these studies has caused considerable debate that will likely be settled following further studies [@doi:10.1111/cen.14276; @doi:10.1016/j.dsx.2020.05.046].
+Overall, the evidence suggest that there is likely a link between low serum 25-hydroxyvitamin D and COVID-19 incidence, these studies must be interpreted with caution as there is the potential for reverse causality, bias, and other confounding factors including that vitamin D deficiency is also associated with numerous pre-existing conditions and risk factors that can increase the risk for severe COVID-19 [@doi:10.3390/nu12051466; @doi:10.1038/s41371-020-00398-z; @doi:10.1111/joim.13149; @doi:10.1007/s40520-020-01677-y].   
+
+While these studies inform us of the potential importance of vitamin D sufficiency and the risk of SARS-CoV-2 infection and severe COVID-19, they fail to conclusively determine whether vitamin D supplementation can therapeutically affect the clinical course of COVID-19.
+In one study, 40 vitamin D deficient asymptomatic or mildly symptomatic participants patients were either randomized to receive 60,000 IU of cholecalciferol daily for at least 7 days (n = 16) or a placebo (n = 24) with a target serum 25-hydroxyvitamin D level >50 ng/ml.
+At day 7, 10 patients achieved >50 ng/ml, and another 2 by day 14.
+By the end of the study, the treatment group had a greater proportion of vitamin D-deficient participants that turned SARS-CoV-2 RNA negative and they had a significantly lower fibrinogen levels potentially indicating a beneficial effect [@doi:10.1136/postgradmedj-2020-139065].
```

Comment:
So I actually found the phrasing in the paper confusing myself. I had wrote tested negative too and changed it because they are insinuating that the vitamin D presence may have been a contributing factor. I think its just a complicated result to explain. 

To answer your question. The treatment group and control group were sometimes discharged from hospital before testing negative because they were only mildly affected. The authors are insinuating that those who were treated with vitamin D tested negative for SARS-CoV-2 in greater proportions than those who did not receive vitamin D by the end of the study or before being discharged. Does that make sense? I only included this study because it is the only study that kind of touches upon the supposed antiviral effects of vitamin D, but the unfortunate thing is that it is a rushed study and with a poor methodology.